users_ldap_groups/__openerp__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "Groups assignment",
-    "version": "1.2",
+    "version": "7.0.1.2.1",
```

Comment:
OK, I have just checked on runbot and there's no problem to keep it like that:

![seleccion_001](https://cloud.githubusercontent.com/assets/7165771/9331181/41f491b6-45be-11e5-94d9-15aead7b4fa5.png)